python/pylibcugraph/setup.py

```diff
@@ -122,7 +122,7 @@ def run(self):
                   os.path.join(os.sys.prefix, "lib")
               ],
               libraries=['cudart', 'cusparse', 'cusolver', 'cugraph', 'nccl',
-                         'cugraph_c'],
+                         'cugraph_c', 'cublas'],
```

Comment:
As our CI system force us to rebuild Python packages in GPU tests stage, we are compiling using CUDA 11.0 and we are linking to `libcusolver.so.10` which does not contain the `libcublas`. We need to explicitly link to `libcublas` to fix this